src/NodeVisitor/EscaperNodeVisitor.php

```diff
@@ -82,6 +90,41 @@ protected function doLeaveNode(Node $node, Environment $env)
         return $node;
     }
 
+    private function unwrapConditional(ConditionalExpression $expression, Environment $env)
+    {
+        // convert "echo a ? b : c" to "a ? echo b : echo c" recursively
+        $expr2 = $expression->getNode('expr2');
+        if ($expr2 instanceof ConditionalExpression) {
+            $expr2 = $this->unwrapConditional($expr2, $env);
+        } else {
+            $expr2 = $this->escapeInlinePrintNode(new InlinePrint($expr2, $expr2->getTemplateLine(), $expr2->getNodeTag()), $env, $this->needEscaping($env));
```

Comment:
`InlinePrint` does not take a node tag as third argument. It only has 2 (this applies to all instantiations)

Reply:
removed